src/modules/Packages.rb

```diff
@@ -2702,6 +2700,16 @@ def format_missing_resolvables(type, list)
         _("These items (%{type}) need to be selected to install: %{list}") % {type: type, list: list}
       end
     end
+
+    def fs_mountpoint(filesystem)
+      filesystem.mountpoints[0] || ""
```

Comment:
Should I add a FIXME here (and a note in installer_hack.md) to revisit this again while implementing subvolumes?